bounties/Code Sample For Airgapped Wallet.md

```diff
@@ -0,0 +1,115 @@
+
+Simplifying this to new developers on the ecosystem will give them a sense of why security is prioritized in applications which hold custody to cryptocurrencies & assets. 
+
+All milestones on this Bounty must be fully reached with synchronization  since with one single feature missing (aka. 1 milestone not being accomplished), it won't work as an airgapped wallet requires all the proposed mechanics to work together:
+1. Generate wallet and store securely (using cryptography in this case)
+2. Sign a transaction and keep the fields complete (i.e. the last ledger sequence is high enough to keep up with ledger progress [+100 advancement])
+3. Transmit the signed transaction blob in a secure channel *without* compromising or connecting the airgapped machine to a network (i.e using a QR code)
+4. Relay the transaction to a rippled node and display the results
+
+### Milestone 1 Details
+>store it securely with a password feature built-in using a cryptography algorithm
+
+Encrypting the keypair on the filesystem should look like this, where the password is used as a key to encrypt/decrypt the wallet creds:
+```py
+        kdf = PBKDF2HMAC(
+            algorithm=hashes.SHA256(),
+            length=32,
+            iterations=100000,
+            salt=salt
+        )
+
+        key = base64.urlsafe_b64encode(kdf.derive(bytes(password.encode())))
+
+        crypt = Fernet(key)
+
+        priv = crypt.encrypt(bytes(seed, encoding='utf-8'))
+        seed = crypt.encrypt(bytes(seed, encoding='utf-8'))
+
+        with open(get_path("/Wallet/seed.txt"), "w") as f:
+            f.write(seed.hex())
+
+        with open(get_path("/Wallet/private.txt"), "w") as f:
+            f.write(priv.hex())
+```
+
+
+### Milestone 3 Details
+>**Efficiently** & **Securely** transfer a signed transaction blob
+
+By "securely", do not compromise the airgapped machine with any unreliable channels which includes wired, bluetooth, or local network solutions.
+
+It is best to use a QR code solution as it only transfers the necessary information from the airgapped machine to the outside party.
+
+Example:
+
+<img src="https://user-images.githubusercontent.com/87929946/196018292-f210a9f2-c5f8-412e-98c1-361a72286378.png" width=20% height=20%>
+(Scan me, it's just a signed_tx blob)
+
+## Helpful links/resources
+
+<!--
+Is there anything else that would be helpful for someone picking up this bounty to know about/reference?
+
+Ex.
+* Are there existing solutions to this problem which would be helpful to learn from?
+* Are there open source projects which can be used as a reference?
+* Are there particularly relevant documentation pages?
+-->
+
+- https://en.wikipedia.org/wiki/Air_gap_%28networking%29
+- https://www.howtogeek.com/687792/the-ultimate-defense-what-is-an-air-gapped-computer/
```

Comment:
I would also add the Discussion post down here so people are sure to read it since it has important context (Sometimes a link can be missed :) )

Reply:
Done.